.eslintrc.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+    "env": {
+        "browser": true
```

Comment:
Don't you also need a key here for "es6"?

Reply:
I tried using 
```
"parserOptions": { 
  "ecmaVersion": 6 
} 
```

but then it asks you ti change ALL of the files

Reply:
I don't understand what you mean. You were running into errors for using `const` when we spoke in person. Don't you need to add `"es6": true` to `"env"`?